dallinger/recruiters.py

```diff
@@ -1192,16 +1192,15 @@ def _descendent_classes(cls):
             yield cls
 
 
-BY_NAME = {}
-for cls in _descendent_classes(Recruiter):
-    BY_NAME[cls.__name__] = BY_NAME[cls.nickname] = cls
-
-
 def by_name(name):
     """Attempt to return a recruiter class by name.
 
     Actual class names and known nicknames are both supported.
     """
+    BY_NAME = {}
```

Comment:
I'd drop the uppercase variable name now that it's not a module-scope semi-constant, but otherwise this looks like a safe change.

Reply:
Thanks for the feedback! Changed variable name to lower-case.